interfaces/builtin/docker_support_test.go

```diff
@@ -190,11 +191,26 @@ func (s *DockerSupportInterfaceSuite) TestInterfaces(c *C) {
 }
 
 func (s *DockerSupportInterfaceSuite) TestAppArmorSpec(c *C) {
+	// no features so should not support userns
+	restore := apparmor_sandbox.MockFeatures([]string{}, nil, []string{}, nil)
```

Comment:
Nitpick, this can be simplified (and all other places this is done)
```suggestion
	restore := apparmor_sandbox.MockFeatures(nil, nil, nil, nil)
```